src/components/maze/Gameboard.tsx

```diff
@@ -84,14 +85,19 @@ export function Gameboard({
     // return `${styles.mazeCell} nonPathColorSet${selectedColorSet} ${backgroundColor}${selectedColorSet}`;
     return `${styles.mazeCell} ${backgroundColor}${selectedColorSet}`;
   }
+
   function getPlayerTileClasses(cell: MazeTileData) {
-    let backgroundImage;
-    if (cell.fight || cell.enemyWon || cell.hasCartel || cell.hasExit) {
-      backgroundImage = 'playerBackgroundEmpty';
-    } else {
-      backgroundImage = 'playerBackgroundElementOnTop';
-    }
-    return `${styles.mazeCell} ${styles.playerCell} ${getPlayerImgDirection()} ${backgroundImage}`;
+    const backgroundImage =
+      cell.fight || cell.enemyWon || cell.hasCartel || cell.hasExit
+        ? 'playerBackgroundEmpty'
+        : 'playerBackgroundElementOnTop';
+
+    return clsx(
```

Comment:
I think that this looks prettier as code but i wouldn't understand where this function comes from until i memorice it. IMO it's better to keep this "ugly" but understandable on the first view for make it easyer to maintain for us and for the future programmers that jump into the project.

Reply:
I assumed `clsx` is a pretty common library constructing className strings conditionally, so it should be understandable, but if you think it's not, I can add a comment or do you think it's better to have older way?

Reply:
If you can document it propperly it should be ok, if not, i think it's better the older way.